File: app/src/main/java/info/nightscout/androidaps/plugins/IobCobCalculator/events/EventIobCalculationProgress.java
package info.nightscout.androidaps.plugins.IobCobCalculator.events;

import info.nightscout.androidaps.events.Event;

public class EventIobCalculationProgress extends Event {
    public String progress;

    public EventIobCalculationProgress(String progress) {
        this.progress = progress;
    }
}

File: app/src/main/java/info/nightscout/androidaps/plugins/iob/iobCobCalculator/events/EventIobCalculationProgress.kt
package info.nightscout.androidaps.plugins.iob.iobCobCalculator.events

import info.nightscout.androidaps.workflow.CalculationWorkflow
import info.nightscout.rx.events.Event

class EventIobCalculationProgress(val pass: CalculationWorkflow.ProgressData, val progressPct: Int, val cause: Event?) : Event()